spec/requests/v1/memberships_spec.rb

```diff
@@ -5,63 +5,63 @@
     it 'returns a list of memberships' do
       supplier1 = FactoryBot.create(:supplier)
       supplier2 = FactoryBot.create(:supplier)
-      user_id = '08223870-9ea3-4250-a14f-9b80ab466afb'
+      user = FactoryBot.create(:user)
```

Comment:
I didn't realise we had a memberships endpoint! this endpoint should probably be removed but happy for that to happen in another PR.

Reply:
OK. Is that a trello card?

Reply:
I personally try and avoid adding Trello cards for tidy up chores like this, especially when they're relatively straightforward as I view this sort of stuff as ongoing maintenance work and clean up we should aim to do as a matter of course without having to prioritise it against feature work. That said, if it's not something that I'm going to be able to do right away and am worried I might forget I have added them as [issues here on GitHub](https://github.com/dxw/DataSubmissionServiceAPI/issues) previously with a hope that I/we would come back to them at some point.